health/micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/PGRDetails.js

```
@@ -185,6 +185,40 @@ const PGRDetails = () => {
+
+        if (isEmpty) {
+          missingFields.push(t(field.label));
+        }
+      }
+    });
+  });
+
+  if (missingFields.length > 0) {
+    setToast({
+      show: true,
+      label: t("CS_COMMON_REQUIRED_FIELDS_MISSING") + ": " + missingFields.join(", "),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)